templates/basic/basic_api.test.js

```diff
@@ -0,0 +1,9 @@
+const request = require("supertest");
+const app = require("./server.js");
```

Comment:
Please follow `ES6`. It's a standard we're following across templates.

Reply:
But in other test files ecma is not followed this is test file for the server
![image](https://github.com/user-attachments/assets/ea3c36ed-7884-471c-a141-fe5a3b676605)
If I convert the test files to follow ES6 then I should change the JEST configurations and other test files to follow the same as well
![image](https://github.com/user-attachments/assets/c05c1ca7-771c-40c4-a94a-31e95d4a6fe8)

Reply:
I can add the jest configurations for the templates dir and run the test for that separately , but if anyone tries to run the test globally it would throw error

Reply:
Hey I'll push a commit to fix this now.